### ISSUE TITLE
Error TS-996002 when build with Ivy - devextreme-angular

### ISSUE THREAD

**Issue by filyjunior** [angular/angular#31472]
# 🐞 bug report

### Description

I just update Angular 8.1.0 from 6.0 and set enableIvy true. Then run "ng build", and I got lots of errors as below.

## 🔬 Minimal Reproduction

enable Ivy in /src/tsconfig.json
ng build

## 🔥 Exception or Error

ERROR in ../node_modules/devextreme-angular/ui/list.d.ts:545:22 - error TS-996002: Appears in the NgModule.imports of TestModule, but could not be resolved to an NgModule class

545 export declare class DxListModule {
~~~~~~~~~~~~
../node_modules/devextreme-angular/ui/toolbar.d.ts:215:22 - error TS-996002: Appears in the NgModule.imports of TestModule, but could not be resolved to an NgModule class

215 export declare class DxToolbarModule {
~~~~~~~~~~~~~~~
../node_modules/devextreme-angular/ui/load-panel.d.ts:260:22 - error TS-996002: Appears in the NgModule.imports of TestModule, but could not be resolved to an NgModule class

## 🌍  Your Environment

Angular CLI: 8.1.0
Node: 10.13.0
OS: win32 x64
Angular: 8.1.0
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router, service-worker

Package Version
@angular-devkit/architect 0.801.0
@angular-devkit/build-angular 0.801.0
@angular-devkit/build-optimizer 0.801.0
@angular-devkit/build-webpack 0.801.0
@angular-devkit/core 0.6.8
@angular-devkit/schematics 0.6.8
@angular/cdk 6.3.2
@angular/http 6.0.6
@angular/material 6.3.2
@angular/pwa 0.6.8
@ngtools/webpack 8.1.0
@schematics/angular 0.6.8
@schematics/update 0.801.0
rxjs 6.5.2
typescript 3.4.5
webpack 4.35.2
devextreme  18.1.4
devextreme-angular 18.1.4

**Comment by jasonaden** [comment]
Thanks for opening the issue. But there's not enough here to be able to actually debug this. You're missing a reproduction, but it's also missing critical information such as whether the upgrade from Angular 6 to Angular 8 worked *without* turning on Ivy mode.

If it did work with version 8 prior to enabling Ivy, it does sound like there might be a compatibility issue with devextreme that should be looked into. But we do need a reproduction in order to look into it. I'll include the auto response below that includes a link where you can create a reproduction:

<hr>

I'm sorry but reported issues require a reproduction of the issue. We suggest using [StackBlitz](https://stackblitz.com/fork/angular-issue-repro2).

If this issue persists, please create a reproduction using one of the links above. Describe the difference between the expected and current behavior in a new issue with the reproduction linked.

**Comment by ovchinnikov** [comment]
Hello @jasonaden , here is an example https://github.com/ovchinnikov/dx-ivy
An upgrade from Angular 6 to Angular 8 works without turning on the Ivy mode. The problem occurs only after the Ivy mode is enabled.
